### ISSUE TITLE
Incorrect backtick imports order

### ISSUE THREAD

**Issue by BraisGabin** [pinterest/ktlint#1445]
## Expected Behavior
If I reformat a file with IntelliJ/AS and then I reformat it with ktlint they don't do/undo the same change.

## Observed Behavior
When I format my code with IntelliJ thse imports are ordered like this:

```kotlin
import org.hamcrest.CoreMatchers.`is`
import org.hamcrest.CoreMatchers.instanceOf
```

But with ktlint they are ordered like this:

```kotlin
import org.hamcrest.CoreMatchers.instanceOf
import org.hamcrest.CoreMatchers.`is`
``` 

## Your Environment
* Version of ktlint used: `0.45.1`

**Comment by paul-dingemans** [comment]
Yes indeed. The backticks are ignored when sorting the imports. This was a conscious change. But I did not knew it conflicts with IntelliJ sorting. I will mark the issue as such.

**Comment by krisgerhard** [comment]
This is an IntelliJ bug. Fixed in 2022.1.
Related GitHub and YouTrack issue: https://github.com/JetBrains/intellij-community/pull/1879

**Comment by paul-dingemans** [comment]
@krisgerhard Tnx for reporting back with reference ;-).
